agent/lib/kontena/launchers/ipam_plugin.rb

```diff
@@ -40,6 +40,14 @@ def image_exists?
       @image_pulled
     end
 
+    def running?
```

Comment:
Is this used anywhere?

Some kind of signal that etcd + kontena-ipam is running would be good, although the Docker/libnetwork timeouts when first attempting to use the network API should cover most of that.